packages/pn-commons/src/utils/costants.tsx

```diff
@@ -1,8 +1,8 @@
-import { FooterLinksType, PreLoginFooterLinksType } from '@pagopa/mui-italia';
+import { FooterLinksType, Languages, PreLoginFooterLinksType } from '@pagopa/mui-italia';
 
 import { getLocalizedOrDefaultLabel } from '../services/localization.service';
 
-export const LANGUAGES = {
+export const LANGUAGES: Languages = {
```

Comment:
The language names are defined differently than in `pn-landing-webapp`. E.g. here it.en = "Inglese', while for `pn-landing-webapp` we have it.en="English".

Reply:
In fact I put a comment where I defined the languages for the landing site under:pn-landing-webapp/src/layout/footer.constants.tsx because I think we need to choose one of the two possible solutions

Reply:
I'm going to change the languages passed into the Footer of the landing site so that it will always show the content translated into the current selected language as decided during today's Standup.